spec/commands/order/all_food_orders_spec.rb

```diff
@@ -29,23 +30,24 @@
     place_order("qwer", "fabien", "burger")
     place_order("poiu", "james", "fish")
 
-    previous_week_order = Order.create(
+    Order.create(
       user_name: "james",
       lunch: "rice",
       date: previous_week
-    )
-    previous_week_order.save
+    ).save
```

Comment:
with `create`, no need to call `save`:

`Order.create(foo: "bar")`

wich is equivalent to `Order.new(foo: "bar").save`

Reply:
Nice catch